codespell_lib/data/dictionary.txt

```diff
@@ -4024,7 +4024,7 @@ breshing->brushing
 brethen->brethren
 bretheren->brethren
 brfore->before
-bridget->bridged
+bridget->bridged, bridge,
```

Comment:
This should be moved to `dictionary_names.txt`

Reply:
Done 7a7c9f4